src/scripts/components/stories/pilot-story/components/02-chapter/02-chapter.module.styl

```diff
@@ -0,0 +1,39 @@
+@require '../../../../../../variables.styl'
+
+.sectionContainer
```

Comment:
> FYI: After rebasing, I haven't figured out how to make it work properly with the `scroll-snap-type: y mandatory` property.

You can add `scroll-snap-align: center` to the outer element of a chapter that should be snapped to